.github/workflows/trigger_pact_docs_update.yml

```diff
@@ -0,0 +1,22 @@
+name: Trigger update to docs.pact.io
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - '**.md'
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger docs.pact.io update workflow
+        run: |
+          curl -X POST https://api.github.com/repos/pact-foundation/docs.pact.io/dispatches \
+                -H 'Accept: application/vnd.github.everest-preview+json' \
```

Comment:
I search for the keyword "application/vnd.github.everest-preview+json" and I think this is quite old. Do we aware about this? https://developer.github.com/changes/3/

Maybe this is new syntax https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-dispatch-event